### ISSUE TITLE
Add support for SHOW VIEWS with LIKE pattern

### ISSUE THREAD

**Issue by AbhishekNew** [prestosql/presto#2999]
Tag as Enhancement

**Comment by AbhishekNew** [comment]
Like

**Comment by electrum** [comment]
You can access view information in bulk from the `information_schema.views` table.

**Comment by AbhishekNew** [comment]
As discussed offline @ebyhr closing this.

**Comment by AbhishekNew** [comment]
Reopening this as I feel should support this as enhancement

**Comment by polaris6** [comment]
@ebyhr 
Is it necessary to implement `SHOW VIEWS` syntax?  Currently Spark and Flink are supported.

if necessary, I am willing to do this, please assign this issue to me if no one else is working on it.

**Comment by findepi** [comment]
Quoting rationale from https://github.com/trinodb/trino/issues/12406 (closed as a duplicate of this one)

> At present, Trino doesn't support the `SHOW VIEWS` syntax, which makes it inconvenient to distinguish between table and view in the table list.

This is correct.
However, if we had two statements: `SHOW TABLES` and `SHOW VIEWS`, a natural expectation would be that the first one shows only tables, and the second one shows only views. This would mean a backwards incompatible change to SHOW TABLES, which IMO is a no-go, so I think we should not add `SHOW VIEWS`.

What we could do, however, is expand output of `SHOW TABLES`, eg include relation type as a second column.

@electrum wdyt?
cc @martint @trinodb/maintainers

**Comment by ebyhr** [comment]
I agree with adding a new field instead of `SHOW VIEWS`. It's similar to MySQL approach. 
https://dev.mysql.com/doc/refman/8.0/en/show-tables.html

**Comment by findepi** [comment]
Thanks @ebyhr for feedback.
Let's give some time for interested @trinodb/maintainers to chime in.

**Comment by hashhar** [comment]
Agreed with @ebyhr . PostgreSQL also does the same.

```
tpch=# \d
         List of relations
 Schema |   Name   | Type  | Owner
--------+----------+-------+-------
 tpch   | customer | table | test
 tpch   | foo      | view  | test
```

**Comment by dain** [comment]
If the table is a materialized view, does it say `materialized view` or just `view`?

**Comment by hashhar** [comment]
It says `materialized view`.

**Comment by subbareddydagumati** [comment]
Any plans to implement this features. It will be helpful.
